kernel/src/hil/acifc.rs

```diff
@@ -0,0 +1,40 @@
+//! Interface for direct control of ACIFC
+//!
+//! Author: Danilo Verhaert <verhaert@cs.stanford.edu>
+
+use returncode::ReturnCode;
+
+/// Enum for selecting which edge to trigger interrupts on.
```

Comment:
Left over commented out code? If so, just remove it.

Reply:
I left this commented out code for now, in case someone wanted to implement an interrupt option. Thought this might be useful for e.g. zero-crossing, but assumed the main use would be simple voltage comparison, so I focused on some basic implementation for that first. You think I should just remove these comments (and leave the implementation for the future)?